src/libs/hooks/useGetCartList.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { AxiosError } from "axios";
+import api from "../api";
+
+interface CartListProps {
+    cartItemsRes: CartItemProps[];
+    orderAmountDetailRes: OrderAmountDetailProps;
+}
+
+export interface CartItemProps {
+    stickerId: number;
+    id: number;
+    mainImageUrl: string;
+    name: string;
+    price: number;
+    discountPrice: number;
+    count: number;
+}
+
+export interface OrderAmountDetailProps {
+    totalAmount: number;
+    productAmount: number;
+    shippingFee: number;
+}
+
+interface CartListResponse {
+    data: CartListProps
+    code: number;
+    message: string;
+}
+
+const useGetCartList = () => {
+    const [response, setResponse] = useState<CartListProps>({} as CartListProps);
+    const [error, setError] = useState<AxiosError>();
+    const [loading, setLoading] = useState(true);
+
+    const fetchData = async () => {
+        await api.get('/cart')
+            .then(res => {
+                const data: CartListResponse = res.data;
+                setResponse(data.data);
+            })
+            .catch(err => {
+                setError(err);
```

Comment:
여기도 에러페이지로 이동하게 해줘야하지 않나용?!
```suggestion
                setError(err);
                navigate('/error')
```